VMEncryption/main/DiskUtil.py

```diff
@@ -1172,3 +1172,14 @@ def get_luks_header_size(self, device_path=None):
                 header_size = None
 
             return header_size
+            
+    def get_osmapper_name(self):
+        osmapper_name = CommonVariables.osmapper_name
+        try:
+            rootfs_device = list(filter(lambda d:d.mount_point=='/',self.get_device_items(None)))
+            if len(rootfs_device)!=0 and rootfs_device[0].type == 'crypt':
```

Comment:
pythonic way is 

`if not len(rootfs_device)` instead of `if len(rootfs_device)!=0 `

Reply:
> pythonic way is
> 
> `if rootfs_device` instead of `if len(rootfs_device)!=0 `